From 59e32e3c220dfb563cd1161e2406b7bc1683fb94 Mon Sep 17 00:00:00 2001
From: Michal Marek <mmarek@suse.com>
Date: Wed, 18 Nov 2015 14:20:33 +0100
Subject: [PATCH 2/2] kabi: Restore kabi in struct iscsi_tpg_attrib
References: bsc#954635
Patch-mainline: Never, kabi

The new member added by
patches.drivers/iscsi-target-Add-tpg_enabled_sendtargets-for-disable.patch
fits into a hole on 64bit and is only used by drivers/target. Before the
patch, the struct looked like

struct iscsi_tpg_attrib {
...
	u8                         t10_pi;               /*    40     1 */

	/* XXX 7 bytes hole, try to pack */

	struct iscsi_portal_group * tpg;                 /*    48     8 */
...
};

Signed-off-by: Michal Marek <mmarek@suse.com>

---
 drivers/target/iscsi/iscsi_target_core.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/target/iscsi/iscsi_target_core.h b/drivers/target/iscsi/iscsi_target_core.h
index bf1f5639fa86..c77782434457 100644
--- a/drivers/target/iscsi/iscsi_target_core.h
+++ b/drivers/target/iscsi/iscsi_target_core.h
@@ -778,7 +778,9 @@ struct iscsi_tpg_attrib {
 	u32			demo_mode_discovery;
 	u32			default_erl;
 	u8			t10_pi;
+#ifndef __GENKSYMS__
 	u32                     tpg_enabled_sendtargets;
+#endif
 	struct iscsi_portal_group *tpg;
 };
 
-- 
2.1.4

